internal/file/watch/option_test.go

```diff
@@ -26,13 +26,6 @@ import (
 	"go.uber.org/goleak"
 )
 
-var (
-	// Goroutine leak is detected by `fastime`, but it should be ignored in the test because it is an external package.
-	goleakIgnoreOptions = []goleak.Option{
-		goleak.IgnoreTopFunction("github.com/kpango/fastime.(*Fastime).StartTimerD.func1"),
-	}
-)
-
```

Comment:
Does this update not affect test, e.g. https://github.com/vdaas/vald/pull/526/files#diff-9709653ad164a1d234b584cfc3aee17aR80 ?

Reply:
I think this update not affect.
There are multiple ignores and it was complicated, so watch_test.go Moved to
https://github.com/vdaas/vald/pull/526/files#diff-067a37d43f4012cebe5f5931eda3a8aaR36-R43

Reply:
this link won't work 😓

Reply:
sorry.
Could you please read this lines??
https://github.com/vdaas/vald/blob/test/internal/file_watch/internal/file/watch/watch_test.go#L35-L42